app/src/main/res/layout/list_item_chore.xml

```diff
@@ -1,23 +1,42 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:orientation="horizontal">
+    android:orientation="horizontal"
+    android:weightSum="1">
 
     <CheckBox
         android:id="@+id/checkbox"
-        android:layout_width="35dp"
-        android:layout_height="35dp" />
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        android:scaleX="1.5"
+        android:scaleY="1.5"
+        android:textSize="20sp"
+        android:layout_marginLeft="10dp"/>
 
     <TextView
         android:id="@+id/name"
-        android:layout_width="match_parent"
+        android:layout_width="239dp"
         android:layout_height="wrap_content"
         android:paddingRight="5dp"
         android:paddingLeft="5dp"
         android:paddingTop="10dp"
         android:paddingBottom="10dp"
-        tools:text="Test"/>
+        android:textSize="20sp"
+        tools:text="Test"
+        android:layout_weight="0.55" />
+
+    <Button
+        android:id="@+id/edit"
+        android:layout_width="59dp"
+        android:layout_height="37dp"
```

Comment:
the height and width are kind of random sizes. it would be better to use wrap content for width and height, maybe even match parent for height if that looks ok, or maybe 35dp height. Then adjust the padding and margins if needed,